functions/zram.bash

```diff
@@ -0,0 +1,35 @@
+check_zram_mounts() {
```

Comment:
Test specific code -> move to `zram.bats`

Reply:
like this ?

Either way,  there's currently trouble with the upstream (or the upstream's upstream in fact, https://github.com/StuartIanNaylor/zram-config uses https://github.com/Izual750/overlayfs-tools/tree/fix_xattr_lib_include).
We need to wait for them to sort that out.
I contacted the authorNeed to